twigs/twigs.py

```diff
@@ -325,6 +326,35 @@ def main(args=None):
         parser_acr.add_argument('--no_code', action='store_true', help=argparse.SUPPRESS)
         parser_acr.add_argument('--sast', action='store_true', help=argparse.SUPPRESS)
 
+        
+        # Arguments required for Azure Container Registry container discovery 
```

Comment:
Please update comment text to "AWS Container Registry"

Reply:
done.